storage/range_raftstorage.go

```diff
@@ -348,8 +348,12 @@ func (r *Range) updateRangeInfo() error {
 		return util.Errorf("failed to lookup zone config for Range %s: %s", r, err)
 	}
 	r.SetMaxBytes(zone.RangeMaxBytes)
-	// TODO(kkaneda): Update configHashes and other fields as well (#1362)?
 
+	// Gossip configs and update configHashes if a lease is held. This is normally unnecessary
```

Comment:
`respCache` is stateless (all of its state is on disk). `ApplySnapshot` will never be called on a leader so `tsCache` will be initialized when/if it becomes leader. So I guess we can just close #1362; `maxBytes` turns out to be the only thing that needs to be manually copied.